src/api/config/routes/webui.rb

```diff
@@ -238,6 +238,10 @@
     post 'project/preview_description' => :preview_description
   end
 
+  controller 'webui/request' do
+    get 'autocomplete_reviewers' => :autocomplete_reviewers, as: 'autocomplete_reviewers'
```

Comment:
I'd rather add it to the already existing section for  `controller 'webui/request' do` at the end of the block at line 341.

But from a higher perspective instead, is there any reason why you didn't implemented it in `requests_filter.rb`?

Reply:
Good catch. I overlooked it